### PR TITLE
Fixed malformed json output from sanitize-log.

### LINKED ISSUE #679: Sanitize log JSON output malformed
The json output format for sanitize-log is outputting malformed json. It appears to be outputting both the normal output format and the json output format. Looking briefly at the code, this appears to be resulting from a bug in `nvme-print.c`.

[Here](https://gist.github.com/gcoakes/1f399c8655eba7a2bec73c7ef9059d08) is the log showing this issue. The first command shows a working version on `v1.9`, the second a broken version on `v1.10.1`, and the third a broken version on `master`.

### PR DESCRIPTION
**Pull request by gcoakes**
Fixes #679

```
[root@FM21V304-DEV09 ~]# git clone https://github.com/gcoakes/nvme-cli
Cloning into 'nvme-cli'...
remote: Enumerating objects: 41, done.
remote: Counting objects: 100% (41/41), done.
remote: Compressing objects: 100% (32/32), done.
remote: Total 8992 (delta 19), reused 18 (delta 9), pack-reused 8951
Receiving objects: 100% (8992/8992), 4.19 MiB | 5.81 MiB/s, done.
Resolving deltas: 100% (6804/6804), done.
[root@FM21V304-DEV09 ~]# cd nvme-cli
[root@FM21V304-DEV09 nvme-cli]# git branch
* master
[root@FM21V304-DEV09 nvme-cli]# git checkout sanitize-log-json-fix
Branch sanitize-log-json-fix set up to track remote branch sanitize-log-json-fix from origin.
Switched to a new branch 'sanitize-log-json-fix'
[root@FM21V304-DEV09 nvme-cli]# make
NVME_VERSION = 1.10.1.33.gf30e
    CC nvme-print.o
    CC nvme-ioctl.o
    CC nvme-lightnvm.o
    CC fabrics.o
    CC nvme-models.o
    CC plugin.o
    CC nvme-status.o
    CC nvme-filters.o
    CC nvme-topology.o
    CC plugins/intel/intel-nvme.o
    CC plugins/lnvm/lnvm-nvme.o
    CC plugins/memblaze/memblaze-nvme.o
    CC plugins/wdc/wdc-nvme.o
    CC plugins/wdc/wdc-utils.o
    CC plugins/huawei/huawei-nvme.o
    CC plugins/netapp/netapp-nvme.o
    CC plugins/toshiba/toshiba-nvme.o
    CC plugins/micron/micron-nvme.o
    CC plugins/seagate/seagate-nvme.o
    CC plugins/virtium/virtium-nvme.o
    CC plugins/shannon/shannon-nvme.o
    CC plugins/dera/dera-nvme.o
    CC util/argconfig.o
    CC util/suffix.o
    CC util/json.o
    CC util/parser.o
    CC nvme
[root@FM21V304-DEV09 nvme-cli]# ./nvme sanitize-log -o json /dev/nvme0n1
{
  "nvme0n1" : {
    "sprog" : 65535,
    "sstat" : {
      "global_erased" : 1,
      "no_cmplted_passes" : 0,
      "status" : "(1) Most Recent Sanitize Command Completed Successfully."
    },
    "cdw10_info" : 4,
    "time_over_write" : 4294967295,
    "time_block_erase" : 174,
    "time_crypto_erase" : 34,
    "time_over_write_no_dealloc" : 0,
    "time_block_erase_no_dealloc" : 0,
    "time_crypto_erase_no_dealloc" : 0
  }
}
```

### CODE REVIEW

**Comment by gcoakes** [comment]
@keithbusch, is there anything more I need to do to validate this for acceptance?

File: nvme-print.c [hunk 1]
```diff
@@ -3672,6 +3672,7 @@ void nvme_show_sanitize_log(struct nvme_sanitize_log_page *sanitize,
 		d_raw((unsigned char *)sanitize, sizeof(*sanitize));
 	else if (flags & JSON)
 		json_sanitize_log(sanitize, devname);
+        return;
```
Comment (keithbusch) [inline nvme-print.c]: Shouldn't this return be within { braces } for the 'else if' case?

**Comment by gcoakes** [comment]
Yes, it definitely should. Too much python.

**Comment by keithbusch** [comment]
I've manually applied the patch, squashing the edits along the way into the single commit.

Thanks for the patch!

**Pull request closed without merging**